src/util/getNormalizedSpectrum.ts

```diff
@@ -184,6 +186,10 @@ export function getNormalizedSpectrum(
         y = rollingMedianBaseline(y, filterOptions).correctedSpectrum;
         break;
       }
+      case 'enforcesorted': {
```

Comment:
can you add a test that goes through here? You called the option `enforceScale` in the types